tests/conversion_integration.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -eux
```

Comment:
Do you need `-x`? Produces a lot of output that is not really interesting when things are successful? Maybe echo what is being tested in `convert_and_verify`?

Reply:
That's mostly a relict from debugging, removed it and replaced by an `echo` with the `ffp-convert` command

Reply:
Thanks! Could you echo to stderr? E.g.

```
echo "hello world" >&2
```

Reply:
done